docs/manual-EN/2.query-language/4.statement-syntax/3.utility-statements/show-syntax.md

```diff
@@ -11,6 +11,8 @@ SHOW CONFIGS [graph|meta|storage]
 
 `SHOW HOSTS` is to list storage hosts registered by the meta server. There are 6 columns: ip, port, status (online/offline), leader partitions count in all spaces, leader partitions count in each space, total partitions count in all spaces.
 
+`SHOW PARTS` lists the partition information with the given SPACE.
```

Comment:
list...with...?

Reply:
`SHOW PARTS` lists the partition information of the given SPACE.